qa/tasks/kafka.py

```diff
@@ -0,0 +1,413 @@
+"""
+Deploy and configure Kafka for Teuthology
```

Comment:
this kafka task seems to conflate two different things: setting up a kafka instance, and running the notification tests

it's probably worth splitting this into two separate tasks. having 'kafka' as a separate task means that we could potentially use the kafka instance for other things. having 'notification tests' as a separate task means that we could potentially run the tests against kafka, amqp, or whatever else we integrate